debian/watch

```diff
@@ -0,0 +1,8 @@
+# See uscan(1) for format
+
+# Compulsory line, this is a version 4 file
+version=4
+
+# Direct Git
+opts="mode=git" https://github.com/named-data/ndn-cxx.git \
```

Comment:
Still using upstream repo. Ignore if this packaging is intended to go there though.